app/src/main/java/com/dkanada/gramophone/util/QueryUtil.java

```diff
@@ -221,6 +222,10 @@ private static void applySortMethod(ItemQuery query, String method) {
             case SortMethod.YEAR:
                 query.setSortBy(new String[]{"ProductionYear"});
                 break;
+            case SortMethod.ADDED:
+                query.setSortBy(new String[]{"DateCreated"});
+                query.setSortOrder(SortOrder.Descending);
```

Comment:
Could you remove this line for now? I have a separate preference for the sort order that I was going to enable at some point so it wouldn't be handled in this helper method.

Reply:
That would be great. This line is only there because there was no way to change the sorting direction, and ascending is not what you usually want here. I removed it.